build/configs/rtl8721csm/cpp_loadable/defconfig

```diff
@@ -24,7 +24,9 @@ CONFIG_TOOLS_DIR="../tools"
 CONFIG_APP_BINARY_SEPARATION=y
 CONFIG_NUM_APPS=5
 # CONFIG_APPS_RAM_REGION_SHAREABLE is not set
-# CONFIG_SUPPORT_COMMON_BINARY is not set
+CONFIG_SUPPORT_COMMON_BINARY=y
```

Comment:
Do we need to support common binary by default?

Reply:
I think it's good to set common binary config as we are using loadable with common binary everywhere else. So, we can test cpp-related things easily on loadable.